app/components/JsonLd.tsx

```
@@ -0,0 +1,16 @@
+
+interface JsonLdProps {
+  data: Record<string, any>;
+}
+
+export default function JsonLd({ data }: JsonLdProps) {
+  return (
+    <Script
+      id={`json-ld-${Math.random().toString(36).substring(7)}`}
+      type="application/ld+json"
+      strategy="beforeInteractive"
+      dangerouslySetInnerHTML={{ __html: JSON.stringify(data) }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)